wordpress-mobile/WordPress-Android
# Issue/10649 material reader post

Fixes #10649 

This PR is based on #11164, which needs to be merged first.

This PR moves reader post details and nested screens to the material theme and adds a dark mode support.

![comparison_reader_post](https://user-images.githubusercontent.com/728822/73129376-fd90f000-3f96-11ea-9d25-c6a18f49bc33.png)

To test:
- Navigate to post details, post comments, and list of users who liked it. Try to like/bookmark posts and make sure everything looks and works ok in both light and dark themes.
The theme can be changed from app settings.

PR submission checklist:
